lib/src/blocs/coin_detail_bloc/coin_detail_state.dart

```diff
@@ -21,9 +21,9 @@ class CoinDetailLoadFailure extends CoinDetailState {
 class CoinDetailLoadInProgress extends CoinDetailState {}
 
 class CoinDetailLoadSuccess extends CoinDetailState {
-  final CoinDetail coinDetail;
+  final CoinDetail? coinDetail;
```

Comment:
can it be null?

Reply:
when I solve the problem I try everything I understand. Because I don't understand it well so I don't know the consequences of it.